lib/paratii.eth.js

```diff
@@ -237,4 +265,12 @@ export class ParatiiEth {
       return this._transferPTI(beneficiary, amount)
     }
   }
+
+  async subscribe (type, options) {
+    this.web3.eth.subscribe(type, function () {
```

Comment:
basically this calls web3.eth.subscribe but suppresses all the errors as well as the return value.. why is it not simply:

     return this.web3.eth.subscribe(type, options)